### ISSUE TITLE
Upgrades to the volume summary preview map

### ISSUE THREAD

**Issue by mradamcox** [mradamcox/loc-insurancemaps#40]
The overview/preview map could use a lot of improvements, this ticket is just to list the easier

Current Default View:
![image](https://user-images.githubusercontent.com/10427268/155597791-5961cbc3-38c3-4443-85bc-b3649c84e64a.png)

Current view when changing the layer order (available to signed in users):
![image](https://user-images.githubusercontent.com/10427268/155604140-3c1c2c67-ae46-49e3-8c2e-1cb39bba22fa.png)

- [x] put inside of a collapsible section, a la #31
- [x] expand to 100% width of page (current 80% width was just so people could scroll past easily on mobile, but the collapsible section should hopefully solve this)
- [x] add basemap switcher (add aerial imagery basemap)
- [x] rename "reference layers" to "key map", re: #21 
- [x] add transparency options for each map layer set.
    - I personally like toggles for this more than sliders, but perhaps both could be added gracefully for a full range of options.
- [x] add reset/full extent button
- [ ] #49
    - remove full screen button once this is done.
- [ ] create disabled version of the draggable layer list shown in the second image above
    - currently a basic Svelte component is used to construct this list and give it its interactivity. However, this component doesn't have a toggle to enable/disable the dragging, meaning that this list can only be shown to registered users who are allowed to rearrange the layer order. It would be better if unauthenticated users could also view this full list.

**Comment by mradamcox** [comment]
Lots of work on this completed.
![image](https://user-images.githubusercontent.com/10427268/156659366-ed0aabf5-8c48-4cec-91ff-0a3799dfd2a3.png)


Regarding the final two items in the wishlist:

I've converted the idea of a standalone volume map to its own ticket--this would still be a nice feature, but the design I ended up with through this ticket works quite well in fullscreen, especially widescreen on mobile devices, so the need for a separate map is much lower now.

The design also handles layer display better than before, so I think it's less important for non-authenticated users to see each layer listed.
